gserver/src/format.ts

```diff
@@ -67,22 +79,28 @@ export function correctIndents(text, indent, settings: Settings) {
                 }
             }
             //Now we should find current line format
-            const format = findFormat(line, settings);
+            const format: ResolvedFormat = findFormat(line, settings);
+            if (format && format.symbol === 'Rule:') {
+                currentIndentation = defaultIndentation;
```

Comment:
This extra bit of logic is because the `FORMAT_CONF` is no longer enough on its own to decide how much to indent e.g. a `Given` - we need to know if we are within a `Rule` (which adds another level) or not.

Reply:
Could we have a rule inside the rule? If not - I think, that it would be better to add some bool var like `commentsMode ` (ex. `ruleMode`) and, if it's true - add rule indentation to all the strings except `Rule` itself.

Reply:
Sorry for late reply - `Rule`s can't be nested no